pearcej/cppds
# Make the exercise visible in 2.3

# Description

I made the exercises visible by moving the <note> tag. All the exercises were on the <note> tag. Because of that, they weren't visible. I also made the answer choices random so that when students refresh, the answer choices for each question will be randomized.
## Related Issue



fix #492 
## How Has This Been Tested?
1. Make changes locally.
2.  See the changes locally.
![image](https://github.com/pearcej/cppds/assets/117699634/2caacb98-d7a9-4196-9d2d-f05a78e4cf87)
